### PR TITLE
SHOP-8-Global-Header

### PR DESCRIPTION
**Pull request by cathydao95**

### CODE REVIEW

File: src/components/Header/Header.module.scss [hunk 1]
```diff
@@ -1 +1,42 @@
+.wrapper {
+  border-bottom: 2px solid var(--color-neutral-400);
```
Comment (scottqmn) [inline src/components/Header/Header.module.scss]: Should be `--color-text-primary`

File: src/components/Header/Header.module.scss [hunk 2]
```diff
@@ -1 +1,42 @@
...
+  font-family: var(--font-title);
+  font-weight: 800;
+}
+.links {
+  display: none;
+  text-transform: capitalize;
+  & .label {
+    text-decoration: none;
+    &.cta {
+      color: var(--color-semantic-error);
```
Comment (scottqmn) [inline src/components/Header/Header.module.scss]: Should be `--color-semantic-action`

File: src/components/Header/Header.module.scss [hunk 3]
```diff
@@ -1 +1,42 @@
...
+    }
+  }
+}
+
+.btn {
```
Comment (scottqmn) [inline src/components/Header/Header.module.scss:42]: Fix alignment by specifying line-height on button 
image

File: src/components/Header/Header.tsx [hunk 4]
```diff
@@ -9,11 +12,51 @@ export const Header: React.FC<HeaderProps> = ({
...
-      <div>primary: {JSON.stringify(primaryLinks)}</div>
-      <div>secondary: {JSON.stringify(secondaryLinks)}</div>
+    <header className={styles.wrapper}>
+      <nav className={styles.navbar}>
+        <button
+          type="button"
+          className={clsx(styles.btn, styles.menuBtn)}
+          onClick={() => console.log("open side bar")}
+        >
+          <FaBars />
```
Comment (scottqmn) [inline src/components/Header/Header.tsx]: Minor but use MUI icons instead

File: src/components/Header/Header.module.scss [hunk 5]
```diff
@@ -1 +1,42 @@
...
+}
 
+.logo {
+  font-size: 1.75rem;
+  font-family: var(--font-title);
+  font-weight: 800;
+}
+.links {
+  display: none;
+  text-transform: capitalize;
```
Comment (scottqmn) [inline src/components/Header/Header.module.scss]: Remove

File: src/components/Header/Header.module.scss [hunk 6]
```diff
@@ -1 +1,42 @@
+.wrapper {
+  border-bottom: 2px solid var(--color-neutral-400);
+}
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 14px 30px;
```
Comment (scottqmn) [inline src/components/Header/Header.module.scss]: Reduce padding for mobile

File: src/components/Header/Header.tsx [hunk 7]
```diff
@@ -9,11 +12,51 @@ export const Header: React.FC<HeaderProps> = ({
...
+        </button>
+
+        <div className={styles.logo}>Shop</div>
+
+        <ul className={clsx(styles.links)}>{primaryContent}</ul>
+
+        <div className={styles.btnContainer}>
+          <ul className={clsx(styles.links)}>{secondaryContent}</ul>
+          <button
```
Comment (scottqmn) [inline src/components/Header/Header.tsx]: button should be part of ul
```
<div>
  <ul>
    {secondaryContent}
    <li><button>
```

**Review by scottqmn — CHANGES_REQUESTED** [review]
Look good so far just have some styling comments

File: src/components/Header/Header.module.scss (hunk 5 above)
Comment (cathydao95) [reply src/components/Header/Header.module.scss]: Hi, I tried to make this change but ended up keeping the display as none to remove them on smaller devices. Is there a better method you were thinking of?

Comment (scottqmn) [reply src/components/Header/Header.module.scss]: That's okay just meant to remove the text-transform

File: package.json [hunk 8]
```diff
@@ -24,6 +24,11 @@
     "test:e2e": "playwright test"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.5",
```
Comment (scottqmn) [inline package.json]: Sorry I meant to use MUI icons from the existing react-icons package: https://react-icons.github.io/react-icons/icons?name=md

Comment (cathydao95) [reply package.json]: oh okay, let me change it back

**Review by scottqmn — CHANGES_REQUESTED** [review]
Can you resolve the conflicts in package-lock? Reinstalling should revert it correctly

File: src/components/Header/Header.tsx [hunk 9]
```diff
@@ -9,11 +12,53 @@ export const Header: React.FC<HeaderProps> = ({
...
+          type="button"
+          className={clsx(styles.btn, styles.menuBtn)}
+          onClick={() => console.log("open side bar")}
+        >
+          <MdMenu />
+        </button>
+
+        <div className={styles.logo}>Shop</div>
+
+        <ul className={clsx(styles.links)}>{primaryContent}</ul>
```
Comment (scottqmn) [inline src/components/Header/Header.tsx:51]: Each link should be wrapped in its own `<li>` so handle that where you're mapping

File: src/components/Header/Header.tsx [hunk 10]
```diff
@@ -9,11 +12,53 @@ export const Header: React.FC<HeaderProps> = ({
...
+
+        <ul className={clsx(styles.linksContainer)}>
+          <li className={clsx(styles.links)}>{secondaryContent}</li>
```
Comment (scottqmn) [inline src/components/Header/Header.tsx]: Same comment as above

File: src/components/Header/Header.module.scss [hunk 11]
```diff
@@ -1 +1,61 @@
...
+}
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 17px 11px;
+}
 
+.menuBtn {
+  font-size: 2rem;
```
Comment (scottqmn) [inline src/components/Header/Header.module.scss]: Use px values for buttons/icons

File: src/components/Header/Header.module.scss [hunk 12]
```diff
@@ -1 +1,61 @@
...
+  }
+
+  .navbar {
+    padding: 16px 40px;
+  }
+
+  .links,
+  .linksContainer {
+    padding-left: 0;
+    text-transform: capitalize;
```
Comment (scottqmn) [inline src/components/Header/Header.module.scss]: Remove this line

**Review by scottqmn — CHANGES_REQUESTED** [review]

File: src/components/Header/Header.module.scss (hunk 12 above)
Comment (cathydao95) [reply src/components/Header/Header.module.scss]: Hi scott, can I get more insight on this? I'm not sure which line exactly or if it's both lines that should be removed, but
if I remove padding-left, there is a 40px padding on the left bc of the ul. Is that what we want to maintain? 

And in terms of text-transform, do we want the links to be lowercase? 

Screenshot 2023-02-07 at 8 01 14 PM

**Review by scottqmn — APPROVED** [review]

**Review by scottqmn — APPROVED** [review]

**Pull request merged**